### PR TITLE
Deployment: `ComposableStablePool` V5

### PR DESCRIPTION
**Pull request by jubeira**
# Deployment: `ComposableStablePool` V5

## Deployment tasks checklist:

- [x] Contract(s) deployed to all target networks / testnets
- [x] TX IDs are properly generated
- [x] Action IDs are generated after deployment (if applicable) 
- [x] Outputs for all target networks are generated, and linked in the task `readme`
- [x] Addresses files are up to date with the new deployment 
- [x] Contracts are verified in every network --> Pool not verified in ZKEVM.
- [x] Deployments `CHANGELOG` is updated
- [x] There are no code changes of any kind

## Issue Resolution

N/A

### CODE REVIEW

File: CHANGELOG.md
Comment (EndymionJkb) [inline CHANGELOG.md:16]: We did use V4, and have marked it deprecated. Shouldn't this retain V4 and add V5 to the end? It's a bit weird to lump two deployments into one release, but it also looks weird to have it deprecated below and not mentioned above.

**Review by EndymionJkb — APPROVED** [review]
Just needs the /outputs linked in the readme (and I had one comment about the changelog). Otherwise LGTM if we can get CI to pass.

File: CHANGELOG.md
Comment (jubeira) [reply CHANGELOG.md]: Sounds good; addressed in https://github.com/balancer/balancer-deployments/pull/61/commits/6a43fd278a48acdce6320b863d97e5db4de4a007.

**Review by EndymionJkb — APPROVED** [review]
Looks good now; just waiting on CI.

**Pull request merged**
